erpnext/loan_management/doctype/loan_repayment/loan_repayment.py

```diff
@@ -265,6 +274,10 @@ def make_gl_entries(self, cancel=0, adv_adj=0):
 		if gle_map:
 			make_gl_entries(gle_map, cancel=cancel, adv_adj=adv_adj, merge_entries=False)
 
+	# def repost_future_loan_interest_accruals(self):
+	# 	future_lias = frappe.db.get_all("Loan Interest Accrual", {"docstatus": 1, "posting_date": (">", self.posting_date)})
+	# 	if future_lias:
+
```

Comment:
Please remove commented code.

Reply:
I just added this to check for test cases, this PR is not complete yet, will uncomment and complete those functions. Added a dont-merge lable